### DISCUSSION TITLE
TF5: Cause of excessive fresh water in b.cesm3_cam041_mom.B1850MOM.ne30_L58_t061.003

### DISCUSSION THREAD

**Discussion by gustavo-marques** [NCAR/amwg_dev#109] [TF5: Coupled SE/MOM6]
The plot below shows that net freshwater flux into the ocean is significantly larger than the control case.

![Screen Shot 2022-04-29 at 2 29 55 PM](https://user-images.githubusercontent.com/11339137/166065219-d73212b8-8c60-4b25-86eb-fd4077d9b423.png)

### Control settings (blue)
CAM6 + FV + L32 + CICE5

### 003 settings  (orange)
cam_dev + SE + L58 + CICE6

### To be investigated
We want to understand the difference between Control and 003 by testing one change at a time. The first test is to switch CICE5 to CICE6 

### New run

CAM6 + FV + L32 + CICE6

CASEROOT: /glade/work/gmarques/cesm.cases/B/bmom.e23.f09_t061_zstar_N65.nuopc.cice6_fv.001

**Comment by cecilehannay** [comment]
Gustavo ran CAM6 + FV + L32 + CICE6 over the week-end. He identified that that configuration had the same issue with excessive freshwater flux and corresponding sea level rise.  This pointed to an issue with CICE6. 

Marika and Dave identified frazil freshwater flux wasn't being accounted for in the CICE6 runs. They provided a bugfix that is currently tested.

**Comment by cecilehannay** [comment]
The CICE6 bugfix fixed the excessive fresh water flux in [CAM6-FV-L32 + MOM6 + CICE6-bugfix]

We have started a new simulation: [CAM_DEV-SE-L58 + MOM6 + CICE6-bugfix]: See https://github.com/NCAR/amwg_dev/issues/115
